### DISCUSSION TITLE
Postgresql DB major release update  like 15.x  to 17.x

### DISCUSSION THREAD

**Discussion by Ramki33** [cloudnative-pg/cloudnative-pg#6080] [Q&A]
Hi, 

I want to know , how to do Postgresql DB major release upgraded like 15.x  to 17.x  pls share links/doc for this .

We deploy PGDB In cluster mode  & upgrade major release , do we have down time? and approximately how long down time will be . 
Is that down time depends on size of the DB  

Is there any possibility to avoid down time ,  its it possible by application continue read & write from DB while another instance in the cluster get upgraded .  rolling update will help to avoid down time or rolling upgrade will support only minor update

**Comment by AntonioMaccarini** [comment]
Hi, 

About the major upgrades there is a milestone for that: https://github.com/cloudnative-pg/cloudnative-pg/issues/4682
